src/index.html

```diff
@@ -1,13 +1,69 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="layout">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body class="layout__content">
+
+    <div data-qa="card" class="layout__item card">
+      <div class="card__product-img-box">
+        <img
+        src="./images/imac.jpeg"
+        class="card__product-img"
+        alt="MAC Computer monitor image"
+        >
```

Comment:
```suggestion
        <img
          src="./images/imac.jpeg"
          class="card__product-img"
          alt="MAC Computer monitor image"
        >
```

[CHECKLIST](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 14